apps/judicial-system/backend/src/app/modules/notification/services/caseNotification/caseNotification.service.ts

```diff
@@ -1332,14 +1388,14 @@ export class CaseNotificationService extends BaseNotificationService {
       arraignmentDate,
     )
 
-    return this.sendEmail(
+    return this.sendEmail({
       subject,
       html,
-      defenderName,
-      defenderEmail,
-      undefined,
-      true,
-    )
+      recipientName: defenderName,
+      recipientEmail: defenderEmail,
+      attachments: undefined,
```

Comment:
Consider removing undefined properties as they are no longer needed when passing in an object as a parameter.